Tests/UWP.Specs/UWP.Specs.csproj

```diff
@@ -87,7 +87,7 @@
       <Version>6.2.14</Version>
     </PackageReference>
     <PackageReference Include="MSTest.TestAdapter">
-      <Version>3.3.1</Version>
+      <Version>3.6.3</Version>
```

Comment:
FYI, I had problems running this project with newer versions of MStest. Haven't found out if it the problem is on my side or the updated package. 
But since we don't run this automatic, I'm fine bumping the version to silence nuget warnings. 
Alternatively we can silence nuget warnings from test projects?

Reply:
I don't have the UWP tools installed myself, so I never had that problem.

Reply:
Just to be sure, I'll role this back.